LibreNMS/Alert/AlertUtil.php

```diff
@@ -36,14 +37,11 @@ class AlertUtil
     /**
      * Get the rule_id for a specific alert
      *
-     * @param  int  $alert_id
      * @return mixed|null
      */
-    private static function getRuleId($alert_id)
+    private static function getRuleId(int $alert_id)
     {
-        $query = 'SELECT `rule_id` FROM `alerts` WHERE `id`=?';
-
-        return dbFetchCell($query, [$alert_id]);
+        return Alert::select('rule_id')->where('id', $alert_id)->get();
```

Comment:
have you tested this? I dont think its correct, since it returns a Collection, and not a value

Reply:
```
>>> Alert::select('rule_id')->where('id', 2)->get();
=> Illuminate\Database\Eloquent\Collection {#5198
     all: [
       App\Models\Alert {#5202
         rule_id: 2,
       },
     ],
   }

>>> Alert::where('id', 2)->value('rule_id');
=> 2

>>>
```

Reply:
No I didn't know how to, now I know Tinker, I'll test. Thanks!